scripts/website.js

```diff
@@ -76,6 +78,48 @@ const tests = [
   ...acquit.parse(fs.readFileSync(path.join(testPath, 'docs/schemas.test.js')).toString())
 ];
 
+function deleteAllHtmlFiles() {
+  try {
+    fs.unlinkSync('./index.html');
+  } catch (err) {
+    if (err.code !== 'ENOENT') {
+      throw err;
+    }
+  }
+  const foldersToClean = [
+    './docs',
+    './docs/tutorials',
+    './docs/typescript',
+    './docs/api',
+    './docs/source/_docs',
+    './tmp'
+  ];
+  for (const folder of foldersToClean) {
+    let files = [];
+    try {
+      files = fs.readdirSync(folder);
+    } catch (err) {
+      if (err.code === 'ENOENT') {
+        continue;
+      }
+    }
+    for (let index = 0; index < files.length; index++) {
+      if (files[index].endsWith('.html')) {
+        fs.unlinkSync(`${folder}/${files[index]}`);
+      }
+    }
+  }
+}
+
+function moveDocsToTemp() {
+  rimraf.sync('./tmp');
+  const folder = `./docs/7.x`;
```

Comment:
this likely should rely on `versionObj` like the website generation, or are there problems with using that?